### DISCUSSION TITLE
Option to auto-fill and not copy to clipboard

### DISCUSSION THREAD

**Discussion by richardj-bsquare** [Authenticator-Extension/Authenticator#621] [Ideas]
**Describe the feature you want:**

Auto-fill works great, how about an option to auto-fill, whilst not copying the code to the clipboard.

**Why do you want this feature in Authenticator?**

It can speed up productivity; sometimes you want to keep what is currently in the clipboard and login; and then paste what's in your clipboard. Or you just want to go look at something half-way through a cut-paste operation... You get the idea...

**Comment by richardj-bsquare** [comment]
Any reason, why just closed, and no discussion?

**Reply by Sneezry** [reply]
It's not closed. We just move non-tech topics from issues to discussions.

**Comment by TriMoon** [comment]
@Sneezry This future is still in need....
If the clipboard can't be eliminated due to technical reasons, then at least RESTORE the clipboard to it's previous content after pasting in the code :wink:
